src/main/scala/net/artsy/atomic/AtomicEventStore.scala

```diff
@@ -181,35 +186,66 @@ abstract class AtomicEventStore[EventType <: Serializable: Scoped: ClassTag, Val
   class Receptionist(
     logProps: String => Props
   ) extends Actor {
-    var logs = Map.empty[String, ActorRef]
+    var logs = Map.empty[String, LogSingleton]
 
-    def liveLogForScope(scope: String): ActorRef = {
+    def liveLogForScope(scope: String): LogSingleton = {
       val (newLogs, targetLog) = logs.get(scope) match {
         case Some(foundLog) => (logs, foundLog)
         case None =>
-          // Recreate the log, which will recall all preexisting events
-          val materializedLog = context.actorOf(logProps(scope), scope)
+          val system = context.system
+          val singletonProps = ClusterSingletonManager.props(
+            singletonProps     = logProps(scope),
+            terminationMessage = PoisonPill,
+            settings           = ClusterSingletonManagerSettings(system)
+          )
+          val manager = system.actorOf(singletonProps, "EventLog-" + scope)
```

Comment:
Will this past tests using `context.actorOf`? If so, I think that would be the more standard approach. If not, we can figure it out later.